### ISSUE TITLE
Locale fails to install

### ISSUE THREAD

**Issue by yochananmarqos** [hongquan/CoBang#31]
I'm attempting to update the AUR package, but it's failing:

```
==> Starting build()...
+ exec meson setup --prefix /usr --libexecdir lib --sbindir bin --buildtype plain --auto-features enabled --wrap-mode nodownload -D b_lto=true -D b_pie=true CoBang-0.7.0 build
The Meson build system
Version: 0.57.1
Source dir: /build/cobang/src/CoBang-0.7.0
Build dir: /build/cobang/src/build
Build type: native build
Project name: cobang
Project version: 0.7.0
Host machine cpu family: x86_64
Host machine cpu: x86_64
Program python3 found: YES (/usr/bin/python3)
Found pkg-config: /usr/bin/pkg-config (1.7.3)
Dependency python found: YES (pkgconfig)
Run-time dependency glib-2.0 found: YES 2.66.7
Run-time dependency gobject-2.0 found: YES 2.66.7
Run-time dependency gobject-introspection-1.0 found: YES 1.66.1
Run-time dependency gtk+-3.0 found: YES 3.24.26
Program desktop-file-validate found: YES (/usr/bin/desktop-file-validate)
Program appstream-util found: YES (/usr/bin/appstream-util)
Program glib-compile-schemas found: YES (/usr/bin/glib-compile-schemas)
Configuring cobang using configuration
Configuring fallback_version.py using configuration
Program flatpak/meson/postinstall.py found: YES (/build/cobang/src/CoBang-0.7.0/flatpak/meson/postinstall.py)
Build targets in project: 3

Found ninja-1.10.2 at /usr/bin/ninja
ninja: Entering directory `build'
ninja: no work to do.
==> Entering fakeroot environment...
==> Starting package()...
ninja: Entering directory `build'
ninja: no work to do.
Installing subdir /build/cobang/src/CoBang-0.7.0/cobang to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/prep.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/__main__.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/ui.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/net.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/messages.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/resources.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/common.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/__init__.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/logging.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/app.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/cobang/consts.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/main.glade to /build/cobang/pkg/cobang/usr/share/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/about.glade to /build/cobang/pkg/cobang/usr/share/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/url-display.glade to /build/cobang/pkg/cobang/usr/share/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/wifi-display.glade to /build/cobang/pkg/cobang/usr/share/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/cobang_128.png to /build/cobang/pkg/cobang/usr/share/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/wireframe-image-add.svg to /build/cobang/pkg/cobang/usr/share/cobang
Installing /build/cobang/src/CoBang-0.7.0/data/vn.hoabinh.quan.CoBang.desktop to /build/cobang/pkg/cobang/usr/share/applications
Installing /build/cobang/src/CoBang-0.7.0/data/vn.hoabinh.quan.CoBang.svg to /build/cobang/pkg/cobang/usr/share/icons/hicolor/scalable/apps
Installing /build/cobang/src/CoBang-0.7.0/data/vn.hoabinh.quan.CoBang.appdata.xml to /build/cobang/pkg/cobang/usr/share/metainfo
Installing /build/cobang/src/CoBang-0.7.0/data/vn.hoabinh.quan.CoBang.gschema.xml to /build/cobang/pkg/cobang/usr/share/glib-2.0/schemas
Installing /build/cobang/src/build/exe/cobang to /build/cobang/pkg/cobang/usr/bin
Installing /build/cobang/src/build/fallback_version.py to /build/cobang/pkg/cobang/usr/lib/python3.9/site-packages/cobang
Running custom install script '/usr/bin/meson --internal gettext install --subdir=po --localedir=share/locale --pkgname=cobang'
Running custom install script '/build/cobang/src/CoBang-0.7.0/flatpak/meson/postinstall.py'
install: cannot stat 'po/fil/LC_MESSAGES/cobang.mo': No such file or directory
y'
install: cannot stat 'po/fil/LC_MESSAGES/cobang.mo': No such file or directory
```

**Comment by yochananmarqos** [comment]
It's fixed with 0.8.0.

**Comment by hongquan** [comment]
Thank you, I don't know why it failed and why 0.8.0 fixed it 😅.
